index.js

```diff
@@ -2,32 +2,40 @@ const Web3 = require("web3");
 const HDWalletProvider = require("@truffle/hdwallet-provider");
 const { getAbi, getAddress } = require("@uma/core");
 const { parseFixed } = require("@ethersproject/bignumber");
+require("dotenv").config();
```

Comment:
Can you document these changes in the README? It also may impact the docs on the docs site.

Reply:
Updated readme to make clear that MNEMONIC should be set as an environment variable